congame-doc/introduction.scrbl

```diff
@@ -1,37 +1,530 @@
+
+@subsection{Upload the study}
+
+Navigate to the Congame server in your web browser and log in with your username and password.
+
+Click on the @onscreen{Admin} tab. (If you don’t see this, contact your server admin.)
+
+Under the @italic{Studies} section, click @onscreen{New Study}, and fill out the upload form as
+follows:
+
+@itemlist[
+
+@item{@bold{Name}: Enter @racketvalfont{Age Survey} (any human-friendly title will do).}
+
+@item{@bold{Slug}: Leave this blank.}
+
+@item{@bold{Type}: Set to @racketresultfont{Conscript}.}
+
+@item{@bold{Study ID}: Enter @racketvalfont{simple-survey}. This is the identifier we used in the 
+@racket[defstudy] and @racket[provide] expressions in our @filepath{age-survey.rkt} file.}
+
+@item{@bold{Study Source}: Click the @kbd{Browse} button and locate/select the
+@filepath{age-survey.rkt} file.}
+
+]
+
+@screenshot["intro-upload.png"]
+
+When ready, click @kbd{Create} to upload and create the study.
+
+@margin-note{If you receive errors at this point, double check that the contents of
+@filepath{age-survey.rkt} exactly match the contents of the example file shown above. In particular,
+check that the top line says @code{#lang conscript} and that the @racket[(provide simple-survey)]
+expression is present in the file.}
+
+Alternatively, if too much time has passed since you clicked the @onscreen{New Study} link, you may
+receive a “session expired” error — simply fill out the form and try again more quickly this time.}
+
+@;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@subsection{Creating a study instance}
+
+As soon as the upload is complete, you’ll be taken to a screen listing all the @tech{study
+instances} for the newly-created study:
+
+@screenshot["intro-empty-instancelist.png"]
+
+Click on @onscreen{New Instance}. For now, just fill in a value like @racketvalfont{Instance 1}
+in the @italic{Name} field, and leave the rest of the fields at their default values:
+
+@screenshot["intro-new-instance.png"]
+
+Click @kbd{Create} to create an instance of the study.
+
+You’ll be taken back to the study’s instance list, and this time the new instance will appear in the
+list.
+
+@;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@subsection{Trying it out}
+
+Now that you’ve published your study and created an instance for it, you can go click on the
+@onscreen{Dashboard} link and see the new instance listed there. 
+
+You’ll also see a link titled @onscreen{Enroll} — click it! You will be taken through the study
+exactly as you were when you tried it out on your computer earlier. Go ahead and complete all the
+steps.
+
+@margin-note{As long as the study instance is in “active” status, other people who log into this same study as
+participants will see this same study instance and the @onscreen{Enroll} link on their dashboard,
+allowing them to complete the study as well.}
+
+Once you’re done with the study, @mark{navigate back to the dashboard somehow}. Then click on the
```

Comment:
Hm, I see, you have to manually change the url to get to the home page. This is somewhat annoying, but we don't want - on purpose - to show the homescreen, since for most participants we don't want that. Would it be helpful if we displayed the home button for the dashboard for admin and researcher users?

Reply:
> Would it be helpful if we displayed the home button for the dashboard for admin and researcher users?

I think that would be great. For everyone else, perhaps the server could auto-add a message at the bottom of any step that transitions to itself that says “You can now close this browser tab/window” (or similar).

Reply:
> For everyone else, perhaps the server could auto-add a message at the bottom of any step that transitions to itself that says “You can now close this browser tab/window” (or similar).

Is this a problem you've had where you got confused by having a button that just goes to itself? In real studies, such buttons should be taken out, so that the final page is simply the final page with whatever custom message the study creator wants.